### PR TITLE
Create toaster component

### LINKED ISSUE #73: Implement notification component

### PR DESCRIPTION
**Pull request by bramvanhoutte**
Closes #73

### CODE REVIEW

File: src/components/Toaster/ToastManager/ToastManager.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  duration: number;
+  onClose: () => void;
+  intent: AlertIntent;
+  isShown: boolean;
+}
+
+export type ToastCallHandler = (message: string, settings: ToastSettings) => void;
+
+interface Props {
+  bindNotify: (handler: ToastCallHandler) => void;
```
Comment (RobinWijnant) [inline src/components/Toaster/ToastManager/ToastManager.tsx:28]: What does notify?

File: src/components/Toaster/ToastManager/ToastManager.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      return updatedToasts;
+    }
+    return toasts;
+  };
+
+  const createToastInstance = (
+    message: string,
+    { isClosable = true, duration = 5, id, intent }: ToastSettings,
+  ): ToastInstance => {
+    const uniqueId = idCounter;
```
Comment (RobinWijnant) [inline src/components/Toaster/ToastManager/ToastManager.tsx:86]: I think keeping state of the latest id can be avoided when using a Date() timestamp with ms as id

File: src/components/Toaster/Toaster.test.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    const button = getByTestId('button-toaster');
+    fireEvent.click(button);
+
+    const toasterElement = getByText(/I hope this closes./);
+
+    const closeIcon = toasterElement.parentElement?.parentElement?.querySelector('.closeAlert');
+    fireEvent.click(closeIcon!);
+
+    await waitFor(() => expect(toasterElement).not.toBeInTheDocument());
```
Comment (RobinWijnant) [inline src/components/Toaster/Toaster.test.tsx:85]: Nice <3

**Review by RobinWijnant — CHANGES_REQUESTED** [review]

File: src/components/Toaster/Toast/Toast.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  ({
+    duration,
+    onRemove,
+    isShown: isShownProp,
+    // Template props
+    intent,
+    message,
+    isClosable,
+  }: Props) => {
+    const [isShown, setIsShown] = useState(true);
```
[the remark this answers is not in the document]
Comment (bramvanhoutte) [reply src/components/Toaster/Toast/Toast.tsx:27]: This was so the ToastManager could close the toasts from the parent.

**Review by RobinWijnant — APPROVED** [review]

**Pull request merged**